libcontainer/cgroups/fs/cpuacct.go

```diff
@@ -120,3 +133,37 @@ func getPercpuUsage(path string) ([]uint64, error) {
 	}
 	return percpuUsage, nil
 }
+
+func getPercpuUsageInModes(path string) ([]uint64, []uint64, error) {
+	percpuUsageKernelMode := []uint64{}
+	percpuUsageUserMode := []uint64{}
+
+	data, err := ioutil.ReadFile(filepath.Join(path, cgroupCpuacctUsageAll))
+	if err != nil {
+		return percpuUsageKernelMode, percpuUsageUserMode, err
+	}
+
+	lines := strings.Split(string(data), "\n")
+	lines = lines[cpuacctUsageAllHeaderLineNo:] //omit header line
+
+	for _, line := range lines {
+		lineFields := strings.Fields(strings.TrimSpace(string(line)))
+		if len(lineFields) == 0 {
```

Comment:
What we really care about is to have at least three columns available.
```suggestion
		if len(lineFields) < 3 {
```